service/src/main/java/org/cbioportal/service/impl/StructuralVariantServiceImpl.java

```diff
@@ -77,6 +86,16 @@
                 if(CollectionUtils.isNotEmpty(sampleIds))  {
                     fusionVariantSampleIds.add(sampleIds.get(i));
                 }
+            } else if (molecularProfileId.endsWith(molecularProfileUtil.MUTATION_PROFILE_SUFFIX)) {
+                MolecularProfile mp = molecularProfileMap.get(molecularProfileId);
+                if (mp.getDatatype().equals(molecularProfileUtil.FUSIONS_AS_MUTATIONS_DATATYPE)) {
+                    String mutationMolecularProfileId = molecularProfileUtil.replaceFusionProfileWithMutationProfile(molecularProfileId);
+                    molecularProfileIdReplaceMap.put(mutationMolecularProfileId, molecularProfileId);
+                    fusionVariantMolecularProfileIds.add(mutationMolecularProfileId);
+                    if(CollectionUtils.isNotEmpty(sampleIds))  {
```

Comment:
add space after 'if'

also, this seems to be handling a case where there is a "_fusions" profile and also a "_mutations" profile of datatype fusions. (by merging the new "_mutations" profile sample ids into the previously found "_fusions" profile sample id.) This seems good, but maybe we should document what combinations of profiles are legal/supported and what combinations are not. Above I said I think the combination of a regular "_mutations" profile plus a fusions datatype "_mutations" profile is not supported. There also may be "_sv" profiles. I can't keep track of what constraints we have when mixing and matching different kinds fo profiles. (or having multiples of a type)